homeassistant/components/advantage_air/sensor.py

```diff
@@ -85,7 +89,7 @@ def icon(self):
     async def set_time_to(self, **kwargs):
```

Comment:
Can you add the type hints here?
`async def set_time_to(self, **kwargs: Any) -> None:`

Reply:
Done